Lib/test/test_compile.py

```diff
@@ -791,6 +792,41 @@ def or_false(x):
                 self.assertIn('LOAD_', opcodes[0].opname)
                 self.assertEqual('RETURN_VALUE', opcodes[1].opname)
 
+    def test_imported_load_method(self):
+        sources = [
+            """\
+            import ast
```

Comment:
Could you use `sys` or `os` instead of `ast` and `concurrent`. No point in making this test slower than necessary.